code/workspaces/client-api/src/dataaccess/datalabRepository.js

```diff
@@ -4,11 +4,11 @@ function Datalab() {
   return database.getModel('Datalab');
 }
 
-function getAll(user) {
+function getAll() { // user
```

Comment:
Is the `// user` comment still meant to be there now there are no arguments?

Reply:
Yes, it is a reminder that the user context has been passed down. This will be used in the near future and will save digging around the code to find it.